mmengine/runner/runner.py

```diff
@@ -1705,6 +1705,19 @@ def train(self) -> nn.Module:
 
         # initialize the model weights
         self._init_model_weights()
+
+        # try to enable fast_conv_bn_eval feature
+        fast_conv_bn_eval = self.cfg.get('fast_conv_bn_eval', None)
+        if fast_conv_bn_eval is not None:
+            from mmengine.model.fast_conv_bn_eval import \
+                turn_on_fast_conv_bn_eval
+            modules = fast_conv_bn_eval
+            if isinstance(modules, str):
+                modules = [modules]
```

Comment:
These lines can be also moved to `turn_on_fast_conv_bn_eval`?

Reply:
Fixed in [4e434fb](https://github.com/open-mmlab/mmengine/pull/1202/commits/4e434fb765099c398eb8cc5991d444d90b39753f).